vuepress/docs/next/tutorials/create/ms/add-access-controls.md

```diff
@@ -164,32 +127,39 @@ Promote the admin user to a full `conference-admin` to reinstate the ability to
 5. Confirm the delete icon is visible 
 6. Confirm a Conference can be successfully deleted from the list
 
-## Notes
-### Realm Roles versus Client Authorities
-This tutorial utilizes authorities. In Keycloak, authorities are roles mapped to a user for a specific client. It is possible to assign higher-level Realm Roles directly to users, e.g. `ROLE_ADMIN`, but this can result in collisions between applications using the same roles.
+### Step 10. Configure the roles in `entando.json`
+Entando can automatically add roles to your client (see [the notes below](#notes) for different client options) when your microservice is deployed.
 
-To implement Realm-assigned roles, the code above must be modified:
-- In the backend, use the annotation `@Secured('ROLE_ADMIN)` or `@PreAuthorize(hasRole('ROLE_ADMIN'))`
-- In the frontend, use `keycloak.hasRealmRole` instead of `keycloak.hasResourceRole` 
+1. Modify the `entando.json` by adding the following line to the `microservices/conference-ms`:
```

Comment:
remove "the" :)

Reply:
think it's correct either way, with or without the

Reply:
it's not. https://english.stackexchange.com/questions/399258/should-i-use-the-before-file-names